http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/RoutesTestJsonSupport.scala

```diff
@@ -0,0 +1,107 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import java.net.URL
+import java.time.Instant
+import java.util.UUID
+
+import org.broadinstitute.dsde.workbench.leonardo.model.Cluster.LabelMap
+import org.broadinstitute.dsde.workbench.leonardo.model.{AuditInfo, Cluster, ClusterImage, ClusterInternalId, DataprocInfo, UserJupyterExtensionConfig}
+import org.broadinstitute.dsde.workbench.leonardo.model.google.{ClusterName, ClusterStatus, IP, Instance, OperationName}
+import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
+import org.broadinstitute.dsde.workbench.model.google.{GcsBucketName, GcsPath, GoogleProject}
+import cats.implicits._
+import spray.json._
+import org.broadinstitute.dsde.workbench.leonardo.model.LeonardoJsonSupport._
+import org.broadinstitute.dsde.workbench.model.google.GoogleModelJsonSupport.{GcsPathFormat => _, _}
+import org.broadinstitute.dsde.workbench.model.WorkbenchIdentityJsonSupport._
+import org.broadinstitute.dsde.workbench.leonardo.model.google.GoogleJsonSupport._
+import org.broadinstitute.dsde.workbench.leonardo.service.ListClusterResponse
+import spray.json.DefaultJsonProtocol
+
+object RoutesTestJsonSupport extends DefaultJsonProtocol{
```

Comment:
This is needed for tests that deserialize json to a `Cluster` object, right?

Reply:
yea..these are only needed in the test. I didn't want to pollute main app code with this.